src/main/java/org/frc5687/chargedup/subsystems/ExtendingArm.java

```diff
@@ -0,0 +1,91 @@
+package org.frc5687.chargedup.subsystems;
+
+import org.frc5687.chargedup.Constants;
+import org.frc5687.chargedup.util.OutliersContainer;
+import org.frc5687.lib.drivers.OutliersTalon;
+import org.frc5687.lib.sensors.HallEffect;
+import org.frc5687.chargedup.RobotMap;
+
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class ExtendingArm extends OutliersSubsystem {
+    private OutliersTalon _talon;
+    private HallEffect _topHall;
+    private HallEffect _bottomHall;
+
+    private final PIDController _extArmController;
```

Comment:
We aren't using WPILib PID controller anymore but the integrated controller on the TalonFX. So we do not need this anymore.